tests/integration/targets/mso_schema_template_anp_epg_selector/tasks/main.yml

```diff
@@ -393,9 +390,6 @@
     epg: ansible_test_1
     selector: selector_2
     expressions:
-      - type: expression_1
-        operator: in
-        value: test
```

Comment:
Same here

Reply:
There are already 6 expressions and we don't need 7 expressions any more